pkg/api/rest/create.go

```diff
@@ -43,11 +43,16 @@ type RESTCreateStrategy interface {
 	// sort order-insensitive list fields, etc.  This should not remove fields
 	// whose presence would be considered a validation error.
 	PrepareForCreate(ctx genericapirequest.Context, obj runtime.Object)
-	// Validate is invoked after default fields in the object have been filled in before
-	// the object is persisted.  This method should not mutate the object.
+	// Validate returns an ErrorList with validation errors or nil.  Validate
```

Comment:
`nil` indicates lack of errors? And that is okay and understood by the caller? I've not seen an implementation that returns nil vs it returns an empty `field.ErrorList`. I think it's mostly so that further callers can append without doing a nil-check, but I want to be very sure that `nil` is okay.